docs/openrpc.json

```diff
@@ -0,0 +1,917 @@
+                "description": "Use 'index' to get an address given its index in the wallet."
+              },
+              {
+                "const": "client",
+                "description": "Use 'client' to let the user select the address he/she wants to provide."
+              }
+            ]
+          }
+        },
+        {
+          "name": "full_path",
+          "summary": "Address full path in a wallet",
+          "description": "Should only be used alongside type 'full_path'.",
+          "schema": {
+            "type": "string"
+          }
+        },
+        {
+          "name": "index",
+          "summary": "Address index in a wallet",
+          "description": "Should only be used alongside type 'index'.",
+          "schema": {
+            "type": "number"
+          }
+        }
+      ],
+      "result": {
+        "name": "address",
+        "schema": {
+          "$ref": "#/components/schemas/Address"
+        }
+      },
+      "errors": [
+        {
+          "$ref": "#/components/errors/DifferentNetworkError"
+        },
+        {
+          "$ref": "#/components/errors/NotImplementedError"
+        },
+        {
+          "$ref": "#/components/errors/PromptRejectedError"
+        }
+      ]
+    },
+    {
+      "name": "htr_getUtxos",
+      "summary": "Returns all UTXOs that meet the filter criteria",
+      "paramStructure": "by-name",
+      "params": [
+        {
+          "$ref": "#/components/contentDescriptors/Network"
+        },
+        {
+          "$ref": "#/components/contentDescriptors/Token"
+        },
+        {
+          "name": "maxUtxos",
+          "summary": "Limit number of UTXOs to be returned",
+          "required": true,
+          "schema": {
+            "type": "number"
+          }
+        },
+        {
+          "name": "onlyAvailableUtxos",
+          "summary": "Selects only UTXOs available to be spent",
+          "required": true,
+          "schema": {
+            "type": "boolean"
+          }
+        },
+        {
+          "name": "filterAddress",
+          "summary": "Selects only UTXOs of the specified address",
+          "required": true,
+          "schema": {
+            "$ref": "#/components/schemas/Address"
+          }
+        },
+        {
+          "name": "authorities",
+          "summary": "Selects only UTXOs with specified authority",
+          "schema": {
+            "type": "number"
+          }
+        },
+        {
+          "name": "amountSmallerThan",
+          "summary": "Selects only UTXOs smaller than specified amount",
+          "schema": {
+            "type": "number"
+          }
+        },
+        {
+          "name": "amountBiggerThan",
+          "summary": "Selects only UTXOs greater than specified amount",
+          "schema": {
+            "type": "number"
+          }
+        },
+        {
+          "name": "maximumAmount",
+          "summary": "Returns only UTXOs smaller or equal than specified amount",
```

Comment:
Is this correct? I think this is the maximum amount when you sum all utxos returned

Reply:
OK: change made: this was indeed wrong. Fixed error.